src/config/backpack/base.php

```diff
@@ -68,11 +68,19 @@
 
     /*
     |--------------------------------------------------------------------------
-    | User Model
+    | Authentication
     |--------------------------------------------------------------------------
     */
 
     // Fully qualified namespace of the User model
     'user_model_fqn' => '\App\User',
```

Comment:
It will not break if we keep `user_model_fqn` (and leave it in the RegisterController). But if you then want to use the option to switch authentication guards, you will have to republish your views. Because the new views will contain `$backpackAuth` or `backpackAuth()`.

So it's not really breaking I guess? You just can't use the new feature straight away.